web-registry/src/routes.tsx

```diff
@@ -75,119 +76,100 @@ const Activity = lazy(() => import('./pages/Activity'));
 const CreateBatch = lazy(() => import('./pages/CreateBatch'));
 const Storefront = lazy(() => import('./pages/Marketplace/Storefront'));
 
-export const routes = createRoutesFromElements(
-  <Route element={<RegistryLayout />}>
-    <Route path="/" element={<Home />} />
-    <Route path="verify-email" element={<VerifyEmail />} />
-    <Route path="add" element={<Additionality />} />
-    <Route path="signup" element={<Signup />} />
-    <Route path="certificate" element={<CertificatePage />} />
-    <Route path="projects/wilmot/admin" element={<Seller />} />
-    <Route path="buyers" element={<BuyersPage />} />
-    <Route path="create-methodology" element={<CreateMethodology />} />
-    <Route
-      // TODO: thould this route be moved to /credit-classes?
-      path="create-credit-class"
-      element={<CreateCreditClassInfo />}
-    />
-    <Route path="land-stewards" element={<LandStewards />} />
-    <Route
-      path="methodology-review-process"
-      element={<MethodologyReviewProcess />}
-    />
-    <Route path="projects" element={<Projects />} />
-    <Route path="projects/:projectId" element={<Project />} />
-    <Route
-      path="post-purchase/:projectId/:walletId/:name"
-      element={<PostPurchase />}
-    />
-    <Route path="ecocredits" element={<KeplrRoute component={Dashboard} />}>
+type RouterParams = {
+  reactQueryClient: QueryClient;
+};
+
+export const getRoutes = ({ reactQueryClient }: RouterParams): RouteObject[] =>
```

Comment:
Ignore the diff below, only this line has changed and also the last one with the router declaration.